.gitignore

```diff
@@ -17,7 +18,6 @@
 /rebuild-apps
 /rebuild-shared
 /rebuild-code-studio
-/i18n/code.org/crowdin.yaml
```

Comment:
If you remove this, will old enlistments stop ignoring i18n/code.org/crowdin.yaml? We might accidentally scoop it up - might as well ignore both locations